doc/install_windows.txt

```diff
@@ -126,6 +78,12 @@ and execute the following from the ``Anaconda Prompt``:
 
     $ conda install theano
 
+The ``rdonnelly`` conda channel provides a precompiled version of PyGPU
+and libgpuarray. You can easily install both through the PyGPU package:
+
+.. code-block:: bash
+
+    $ conda install -c rdonnelly pygpu
```

Comment:
Once again, the rdonelly channel is not one we want to enshrine in the docs.  It is only temporary until those pacakages get moved to the default channel.